example/mono_pkg.yaml

```diff
@@ -2,6 +2,7 @@
 dart:
   - dev
   - stable
```

Comment:
Consider removing stable?

Reply:
soon enough we'll have 2.3.0-dev, and we'll want this back

Reply:
why? Testing the minimum (2.0.0) and maximum (dev) should be sufficient.

Maybe current stable on the cron job or something, but not every PR imo.

Reply:
Ah...decent point.